src/tree/localProject/LocalFunctionsTreeItem.ts

```diff
@@ -64,6 +72,37 @@ export class LocalFunctionsTreeItem extends FunctionsTreeItemBase {
             return super.compareChildrenImpl(item1, item2);
         }
     }
+
+    /**
+     * .NET Isolated projects don't have typical "function.json" files, so we'll have to ping localhost to get functions (only available if the project is running)
+     */
+    private async getChildrenForIsolatedProject(context: IActionContext): Promise<AzExtTreeItem[]> {
+        if (runningFuncTaskMap.has(this.parent.workspaceFolder)) {
+            const functions = await requestUtils.sendRequestWithExtTimeout({
+                url: `${this.parent.hostUrl}/admin/functions`,
+                method: 'GET'
+            });
+            return await this.createTreeItemsWithErrorHandling(
+                <WebSiteManagementModels.FunctionEnvelope[]>functions.parsedBody,
+                'azFuncInvalidLocalFunction',
+                async func => {
+                    func = requestUtils.convertToAzureSdkObject(func);
+                    return LocalFunctionTreeItem.create(this, nonNullProp(func, 'name'), func.config, undefined, func, context);
+                },
+                func => func.name
```

Comment:
Would it be worth to mention to start debugging to view this?  I imagine that this would only throw an error is func.name were to be `undefined`.  Could the user potentially kill their func task without the event handler catching it?  Like if they closed it via their terminal or process manager?